internal/api/api.go

```diff
@@ -151,6 +152,11 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, userDeviceSvc
 	udOwner.Get("/offers", valuationsController.GetOffers)
 	udOwner.Get("/instant-offer", valuationsController.GetInstantOffer)
 
+	vOwner := v1Auth.Group("/vehicles/:tokenId", deviceMw)
+	vOwner.Get("/valuations", vehiclesController.GetValuations)
+	vOwner.Get("/offers", vehiclesController.GetOffers)
+	vOwner.Get("/instant-offer", vehiclesController.GetInstantOffer)
```

Comment:
let's change this to be a POST (feel it makes more sense) and then rename method to `RequestInstantOffer`